### DISCUSSION TITLE
`intro` manpage

### DISCUSSION THREAD

**Discussion by Yonle** [termux/termux-packages#9363] [Ideas]
Some Linux Distributions, or BSDs include `intro` manpage for introduction to user commands.

![Screenshot_2022_0311_162819](https://user-images.githubusercontent.com/63401712/157840093-2d2f2c87-4993-409d-bc08-2692e29ee785.png)

Commonly useful for some beginners who's new in Termux, Linux or UNiX.

**Comment by sylirre** [comment]
Termux equivalent is `man termux` and Linux commands page is still `man intro` (needs `manpages` package).

**Comment by 2096779623** [comment]
And one more: https://github.com/termux/termux-packages/blob/master/packages/termux-tools/termux.1.md.in .
